lib/pipeline.js

```diff
@@ -499,6 +487,50 @@ class PipelineModel extends BaseModel {
         return admin;
     }
 
+    async getFirstAdmin() {
+        /* eslint-disable no-restricted-syntax */
+        for (const username of Object.keys(this.admins)) {
+            try {
+                /* eslint-disable no-await-in-loop */
```

Comment:
can replace with `// eslint-disable-next-line no-await-in-loop` and delete line 496

Reply:
I fixed. Thank you!